test/mongodb.test.js

```diff
@@ -211,7 +220,7 @@ describe('mongodb connector', function() {
         /* eslint-disable camelcase */
         var indexes =
         { _id_: [['_id', 1]],
-          geometry_2dsphere: [['geometry', '2dsphere']],
+          geojson_location_geometry: [['location.geometry', '2dsphere']],
```

Comment:
IIUC, this change means our tests are no longer verifying `geometry_2dsphere`. I think we should preserve them, to ensure backwards compatibility is preserved. (This applies to the change above too.)

@superkhau thoughts?